ArduCopter/inertia.cpp

```diff
@@ -21,7 +21,12 @@ void Copter::read_inertia()
         current_loc.alt = inertial_nav.get_altitude();
     } else {
         // with inertial nav we can update the altitude and climb rate at 50hz
-        current_loc.alt = pv_alt_above_home(inertial_nav.get_altitude());
+        float alt;
+        if (pv_alt_above_home(inertial_nav.get_altitude(), alt)) {
+            current_loc.alt = alt;
+        } else {
+            current_loc.alt = 0.0f; // retain compatability with inertial_nav
```

Comment:
Shouldn't this be `inertial_nav.get_altitude()`? Also, if `pv_alt_above_home` checks that home is set (as I suggested in previous comment), then we don't need the if above.